src/js/routes/Ballot/Ballot.jsx

```diff
@@ -214,13 +230,13 @@ export default class Ballot extends Component {
       wait_until_voter_sign_in_completes: wait_until_voter_sign_in_completes,
     });
 
-    if (this.props.location && this.props.location.hash){
+    if (this.props.location && this.props.location.hashvoterGuidesForId) {
```

Comment:
Hi @SailingSteve, since this is a prop (as opposed to a local state), shouldn't it still be "this.props.location.hash" instead of "this.props.location.hashvoterGuidesForId"?

Reply:
yes, that is a complete typo.